### DISCUSSION TITLE
Question - Upgrade best pratctices

### DISCUSSION THREAD

**Discussion by JTrotta** [akkadotnet/akka.net#5947] [Q&A]
What are the best techniques/practises for upgrading the AKKA.Net libraries when a cluster is in production?

**Comment by Aaronontheweb** [comment]
@JTrotta I will get back to you on this as fast as I can - sorry for the delay, was out on Holiday last week

**Reply by JTrotta** [reply]
Thank you for reply @Aaronontheweb . I look forwarding your answer
